### PR TITLE
FizzBuzz in Cobol

### PR DESCRIPTION
**Pull request by UbiquitousBear**

### CODE REVIEW

File: fizzbuzz.CBL [hunk 1]
```diff
@@ -0,0 +1,31 @@
+IDENTIFICATION DIVISION.
```
Comment (iolivia) [inline fizzbuzz.CBL]: I ran this and noticed the output looks a bit weird on the numbers. Can we make those just 1, 2, etc.? 

```
$main
001
002
Fizz
004
Buzz
Fizz
007
008
```

File: fizzbuzz.CBL [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+DATA DIVISION.
+WORKING-STORAGE SECTION.
+01 IV PIC 999 VALUE 1.
+01 FZ PIC 999 VALUE 1.
+01 BZ PIC 999 VALUE 1.
+
+PROCEDURE DIVISION.
+FIZZBUZZ-MAIN SECTION.
+   PERFORM 100 TIMES
+           IF FZ = 3
```
Comment (iolivia) [inline fizzbuzz.CBL]: can this be an AND instead of nested ifs? probably a little more readable.

File: fizzbuzz.CBL (hunk 1 above)
Comment (iolivia) [inline fizzbuzz.CBL]: Can you add COBOL to the readme as well? 😄

**Review by iolivia — COMMENTED** [review]
looks good, a few minor comments 😄 thank you!

File: fizzbuzz.CBL (hunk 1 above)
Comment (UbiquitousBear) [reply fizzbuzz.CBL]: The storage picture doesn't allow for 1 and 11 to coexist in the same data storage, I'll change it to `99` from `999`. I'll have a quick refresh to see if it's actually possible to do (should be, right?)

**Comment by tonyoz93** [comment]
this code is functional?

**Comment by iolivia** [comment]
@tonyoz93 I tested in an online console and it was working, if that's what you mean 😄

**Pull request closed without merging**